subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,85 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+DT_COMPAT_NORDIC_NRF21540_GPIO := nordic,nrf21540_gpio
+DT_COMPAT_SKYWORKS_SKY66112_11 := skyworks,sky66112-11
+
+config MPSL_FEM
+	bool "Radio front-end module (FEM) support"
+	default $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540_GPIO)) || \
+			$(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_SKYWORKS_SKY66112_11))
+	help
```

Comment:
It think we cannot do that. As long as flash driver compatible with MPSL depends on `BT_CTLR` we cannot remove `BT_CTLR` from build when using PA/LNA from MPSL.
After adding flash driver compatible with MPSL to Zephyr tree (I think this work is in progress), we could add such negative dependency to Kconfig

Reply:
This is a bit more complicated, since if you want to have MPSL plus Flash access, then you have to have softdevice controller linked in. If we explicitl deny it as you suggested, then no customer shall be able to make a product with FEM and Flash access.

Reply:
Thanks for providing this insight. I understand that we have to leave it as is for now.. If you're not aware of it, there is already a draft PR open to resolve this odd dependency: https://github.com/tejlmand/fw-nrfconnect-nrf/pull/2

Reply:
I added `depends on !BT` in `config MPSL_FEM_NRF21540_GPIO` only. Tf I am not mistaken then there is no such requirement for `SKY66112` because it has a bypass feature, so it will not block BLE even if not properly controlled. Do you think its ok or should this apply to all types of FEM after all?

Reply:
@rugeGerritsen Is this OK?

Reply:
Unless there is a use case where you would like to have FEM for 802.15.4 but not for Bluetooth LE, I would rather add `depends on !BT` to make it clear that it is not yet supported